### ISSUE TITLE
Error opening CSV files with fewer than 10 rows

### ISSUE THREAD

**Issue by twopoint718** [saulpw/visidata#177]
This CSV file (9 rows) causes a `StopIteration` error:

```csv
name,type,organization,role,email,position,phone
Member1,Member,"Alt Corp","Sales Director",member@example.com,Member,5555555
Member1,Member,"Alt Corp","Sales Director",member@example.com,Member,5555555
Member1,Member,"Alt Corp","Sales Director",member@example.com,Member,5555555
Member1,Member,"Alt Corp","Sales Director",member@example.com,Member,5555555
Member1,Member,"Alt Corp","Sales Director",member@example.com,Member,5555555
Member1,Member,"Alt Corp","Sales Director",member@example.com,Member,5555555
Member1,Member,"Alt Corp","Sales Director",member@example.com,Member,5555555
Member1,Member,"Alt Corp","Sales Director",member@example.com,Member,5555555
```

And this one (10 rows) works fine:

```csv
name,type,organization,role,email,position,phone
Member1,Member,"Alt Corp","Sales Director",member@example.com,Member,5555555
Member1,Member,"Alt Corp","Sales Director",member@example.com,Member,5555555
Member1,Member,"Alt Corp","Sales Director",member@example.com,Member,5555555
Member1,Member,"Alt Corp","Sales Director",member@example.com,Member,5555555
Member1,Member,"Alt Corp","Sales Director",member@example.com,Member,5555555
Member1,Member,"Alt Corp","Sales Director",member@example.com,Member,5555555
Member1,Member,"Alt Corp","Sales Director",member@example.com,Member,5555555
Member1,Member,"Alt Corp","Sales Director",member@example.com,Member,5555555
Member1,Member,"Alt Corp","Sales Director",member@example.com,Member,5555555
```

The error seems to come from this hard coded 10 in the source:

https://github.com/saulpw/visidata/blob/231994e35420c5aab69fb964caccdcf8c847eeaa/visidata/loaders/csv.py#L38

**Comment by saulpw** [comment]
Nice catch, @twopoint718, and thanks for the bug report!  Fixed in 45a34a and will be in v1.3!

**Comment by twopoint718** [comment]
Great! Thanks for the software it’s exactly what I’ve always wanted
